recipes/libprotobuf-mutator/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+find_package(libprotobuf-mutator REQUIRED)
```

```suggestion
find_package(libprotobuf-mutator REQUIRED CONFIG)
```

cmake_find_package_multi provides cmake config file